beacon-chain/slasher/params.go

```diff
@@ -0,0 +1,130 @@
+package slasher
+
+// Parameters for slashing detection.
```

Comment:
Should this be under /shared/params?  Will external entities (e.g. validators) ever need this?

Reply:
These are internal to slasher, no need to modify in the validator or use anywhere else

Reply:
Given internal to slasher, should these be still be exported?